config/vufind/RecordTabs.ini

```diff
@@ -39,6 +39,13 @@ tabs[Preview] = preview
 tabs[Details] = StaffViewArray
 defaultTab = null
 
+[VuFind\RecordDriver\ProQuestFSG]
+tabs[Description] = Description
+tabs[UserComments] = UserComments
+tabs[Reviews] = Reviews
+tabs[Details] = StaffViewMARC
```

Comment:
@sturkel89 reports:
> Staff View tab is not formatted
The contents of the Staff View tab are displayed as a giant scary blob. Demian says that to fix it, you just need to change StaffViewArray to StaffViewMARC in RecordTabs.ini.

Well yes, that was much too easy.  I will miss the giant scary blob.